### PR TITLE
chore(all): address code issues

### PR DESCRIPTION
**Pull request by biffgaut**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 3b1d9a8545e3e17035a9c0edd15a142c3f4b7b15
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=62d858e2-c9b2-461e-b371-2ecdfebe96ff%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by aws-solutions-constructs-team** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: githubautobuild-for-cdk-v2
* Commit ID: 30314b9e43d2855f5814098c9c501a3a2eac6765
* Result: SUCCEEDED
* [Build Logs](https://t2ulkoluaf.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=31249831-fc81-4748-909a-16533af154cf%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts [hunk 1]
```diff
@@ -132,10 +132,12 @@ export class Route53ToApiGateway extends Construct {
...
-    // Create A Record in custom domain to route traffic to API Gateway
-    new route53.ARecord(this, 'CustomDomainAliasRecord', {
+    const arecordProps: route53.ARecordProps = {
       zone: this.hostedZone,
       target: route53.RecordTarget.fromAlias(new targets.ApiGateway(this.apiGateway))
-    });
+    };
+
+    // Create A Record in custom domain to route traffic to API Gateway
+    new route53.ARecord(this, 'CustomDomainAliasRecord', arecordProps); // NOSONAR
```
Comment (georgebearden) [inline source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts:141]: Just for my understanding - this was a sonarqube finding that is being addressed?

File: source/yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,89 @@
+Arguments:
```
Comment (georgebearden) [inline source/yarn-error.log:1]: Is this file intended to be included?

File: source/patterns/@aws-solutions-constructs/core/lib/eventbridge-helper.ts [hunk 3]
```diff
@@ -47,5 +47,6 @@ export function buildEventBus(scope: Construct, props: BuildEventBusProps): even
     const _eventBusName = props.eventBusProps.eventBusName || 'CustomEventBus';
     return new events.EventBus(scope, _eventBusName, props.eventBusProps);
   }
-  return;
+  // ESLint requires this return statement, so disabling SonarQube warning
+  return; // NOSONAR
```
Comment (georgebearden) [inline source/patterns/@aws-solutions-constructs/core/lib/eventbridge-helper.ts:51]: Fun! the tooling yelling at each other

**Review by georgebearden — APPROVED** [review]
Looks good to me - just a few questions.

File: source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts (hunk 1 above)
Comment (biffgaut) [reply source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts:141]: Yes, SonarQube thought this object was ignored since it is not assigned to anything, so wanted me to remove the instantiation.

File: source/yarn-error.log (hunk 2 above)
Comment (biffgaut) [reply source/yarn-error.log:1]: No, thanks

File: source/patterns/@aws-solutions-constructs/core/lib/eventbridge-helper.ts (hunk 3 above)
Comment (biffgaut) [reply source/patterns/@aws-solutions-constructs/core/lib/eventbridge-helper.ts:51]: Fun is one of the adjectives that could be applied...
